tests/spec/s-function.js

```diff
@@ -1,7 +1,177 @@
 /* global describe, it, expect, beforeEach */
+var hasStrictMode = (function () {
+  'use strict';
+
+   return !this;
+}());
+
+var global = (function () {
+   return this;
```

Comment:
Ok, I'll use your suggestion but I have a feeling I read somewhere that this can call problems in some versions of Chrome, but I'm not sure.